jerry-core/jcontext/jcontext.h

```diff
@@ -131,17 +131,14 @@ typedef struct
 #ifdef JMEM_STATS
   jmem_heap_stats_t jmem_heap_stats; /**< heap's memory usage statistics */
 #endif /* JMEM_STATS */
-} jerry_context_t;
 
 #ifndef CONFIG_ECMA_LCACHE_DISABLE
-/**
- * Hash table for caching the last access of properties.
- */
-typedef struct
-{
-  ecma_lcache_hash_entry_t table[ECMA_LCACHE_HASH_ROWS_COUNT][ECMA_LCACHE_HASH_ROW_LENGTH];
-} jerry_hash_table_t;
+  ecma_lcache_hash_entry_t lcache[ECMA_LCACHE_HASH_ROWS_COUNT][ECMA_LCACHE_HASH_ROW_LENGTH]; /**< Hash table for
+                                                                                              *   caching the last
+                                                                                              *   access of properties.
+                                                                                              */
```

Comment:
I would intorduce a typedef for this or move the comment the line before. Also add a comment that this must be the last member of the context for performance reasons.

Reply:
I'll move the comment, typedef would be an overkill for this.